akka-persistence/build.sbt

```diff
@@ -21,3 +21,5 @@ MimaKeys.previousArtifact := akkaPreviousArtifact("akka-persistence-experimental
 fork in Test := true
 
 javaOptions in Test := MultiNode.defaultMultiJvmOptions
+
+
```

Comment:
Kind of, a "mock change" in persistence to sanity check it all works as expected (triggers everything that uses persistence) on jenkins.